dashboard/app/models/professional_learning_course.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: professional_learning_courses
+#
+#  id         :integer          not null, primary key
+#  name       :string(255)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class ProfessionalLearningCourse < ActiveRecord::Base
+  has_many :learning_modules, class_name: 'LearningModule', dependent: :destroy
+  has_many :artifacts, through: :learning_modules, dependent: :destroy
+  has_many :user_course_enrollments, dependent: :destroy
+  #Can I do this through artifacts?
+
+  def self.create_course_with_modules name
```

Comment:
This appears to be a test only method, which the name did not make obvious.  (I expected this was a method I would want to call.)  Please move to a test file.